database/app/src/main/java/com/google/firebase/referencecode/database/MainActivity.java

```diff
@@ -29,32 +30,35 @@
 public class MainActivity extends AppCompatActivity implements MainActivityInterface {
 
     private static final String TAG = "MainActivity";
+    DatabaseReference myRef;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
+        FirebaseDatabase database = FirebaseDatabase.getInstance();
+        myRef = database.getReference("message");
     }
 
     @Override
-    public void basicReadWrite() {
-        // [START write_message]
-        // Write a message to the database
-        FirebaseDatabase database = FirebaseDatabase.getInstance();
-        DatabaseReference myRef = database.getReference("message");
-
-        myRef.setValue("Hello, World!");
-        // [END write_message]
-
+    public void valueRead() {
         // [START read_message]
         // Read from the database
+        /*
```

Comment:
I appreciate the improvement you're trying to make, but I think this comment is too much.  Could you remove it?  We should be covering this material in the documentation that surrounds the snippets.